avocado/utils/asset.py

```diff
@@ -317,8 +317,7 @@ def _has_valid_hash(cls, asset_path, asset_hash=None):
         If asset_hash is None then will consider a valid asset.
         """
         if asset_hash is None:
-            LOG.warning("No hash provided. Cannot check the asset file"
-                        " integrity.")
+            LOG.info("No hash provided. Cannot check the asset file integrity.")
```

Comment:
Actually, the right way to avoid mixing Avocado log messages with test's logs, would be to change the namespace of `LOG` here to `__name__` , instead of using `avocado.test`. This goes on the same line has the recent changes on avocado-vt side. There is an open issue for that: #4997.

I just created a PR (#5121) with just this small change, please, could you try that one and let us know if it will solve your problem?

Reply:
@beraldoleal thanks nice approach there , closing this one